language/move-vm/runtime/src/interpreter.rs

```diff
@@ -67,6 +67,8 @@ pub(crate) struct Interpreter {
     operand_stack: Stack,
     /// The stack of active functions.
     call_stack: CallStack,
+    /// Whether to perform runtime type safety checks.
+    strict: bool,
```

Comment:
strict is somehow not the right notion for this. Both the verifier and the interpreter are strict. Perhaps called it `checked` here and elsewhere?